homeassistant/components/alarmdecoder/binary_sensor.py

```diff
@@ -73,14 +73,16 @@ def __init__(
         relay_chan,
     ):
         """Initialize the binary_sensor."""
-        self._zone_number = int(zone_number)
```

Comment:
It would be cleaner to keep `self._zone_number` and add the `_attr_extra_state_attributes`

Reply:
Okay, I will push this change :+1:

Reply:
@bdraco, done!